examples/aei_from_events/aei_from_events.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+
+all_cat = ['festivals', 'school-holidays', 'airport-delays', 'sports',
+           'performing-arts', 'concerts', 'public-holidays', 'community',
+           'observances', 'severe-weather', 'expos', 'conferences',
+           'health-warnings', 'academic', 'disasters', 'politics', 'terror']
+
+nonattended = ['public-holidays', 'observances', 'school-holidays', 'academic']
```

Comment:
please use snake case and uppercase, NON_ATTENDED.

Reply:
As you wish!